sphinx/tutorial/cylc/furthertopics/family-triggers.rst

```diff
@@ -0,0 +1,184 @@
+Family triggers
+===============
+
+This tutorial walks you through using family :term:`triggers <task trigger>`.
+
+Family triggers allow you to build dependencies based on sets of tasks
+sharing the same namespace, referred to as
+:ref:`families <tutorial-cylc-families>`.
+
+
+Purpose
+-------
+
+Family triggers are used to specify a condition to be met by a set of tasks
+in a particular :term:`family` that will :term:`trigger <task trigger>`
+the next task(s) in the :term:`graph`.
+
+Like regular task triggers, family triggers can depend on the success or
+failure of tasks. However, because a family can contain multiple tasks, you
```

Comment:
I think it would be better to explicitly mention dependence on the *qualifiers* of the family of tasks - not just success or failure as examples. Also it would be useful to reference the 'Cylc Tutorial -> Scheduling -> Further Scheduling -> Qualifiers' section here.